docs/general/installation/container.md

```diff
@@ -24,6 +24,12 @@ Additionally, there are several third parties providing unofficial container ima
 
 [Docker](https://www.docker.com/) allows you to run containers on Linux, Windows and MacOS.
 
+:::warning
+
+While it is possible to run Docker on Windows or macOS, it is an **UNSUPPORTED** configuration for Jellyfin. Please install Jellyfin natively if you wish to use Windows or macOS.
```

Comment:
I wouldn't say it's unsupported as docker/podman work fine on these platforms. The only pain point would be hardware acceleration.

Reply:
Maybe "not recommended" instead? Curious, is this something that comes up a lot and requires a disclaimer, or is it just an opinion thing?